xla/service/gpu/model/hlo_op_profiles.cc

```diff
@@ -21,22 +21,21 @@ limitations under the License.
 #include <variant>
 
 #include "absl/memory/memory.h"
+#include "absl/strings/numbers.h"
```

Comment:
Could you also add:

```
#include <cstdint>

#include "absl/log/check.h"
#include "absl/log/log.h"
```

(linter warning, for `int32_t`, `CHECK()` and `VLOG()`)

Reply:
done